### DISCUSSION TITLE
Various marginal likelihood estimators

### DISCUSSION THREAD

**Discussion by ben18785** [pints-team/pints#1584] [Ideas/Science]
- [Linked importance sampling](https://arxiv.org/pdf/math/0511216.pdf) by Radford Neal.
- [Power posteriors](https://pdfs.semanticscholar.org/14b8/09db5399289b947ef9460c60d3a99d6efb4b.pdf) by Friel.

**Comment by MichaelClerx** [comment]
Linked to #637 ?
